docs/50_eos-evm/30_developer-guide/30_deploy-local-nodes.md

```diff
@@ -860,9 +860,9 @@ Example output:
 }
 ```
 
-### 5. Start TrustEVM-node (a.k.a. Silkworm Node)
+### 5. Start EOS EVM Node (a.k.a. Silkworm Node)
 
-A TrustEVM-node is a node process of the virtual ethereum blockchain that validates virtual ethereum blocks and serves the read requests coming from TrustEVM-RPC. It will not produce blocks. However, it will consume blocks from Antelope node and convert Antelope blocks into Virutal Ethereum blocks in a deterministic way.
+A EOS EVM Node is a node process of the virtual ethereum blockchain that validates virtual ethereum blocks and serves the read requests coming from EOS EVM RPC. It will not produce blocks. However, it will consume blocks from Antelope node and convert Antelope blocks into Virutal Ethereum blocks in a deterministic way.
 
 To set it up, we need to first make up a genesis of the virtual ethereum blockchain that maps to the same EVM state of the evm account of the Antelope chain that just initialized in the previous steps.
```

Comment:
we need to first -> first we need to
ethereum -> Ethereum
evm account -> EVM account